apps/web/lib/integrations/slack/transform.ts

```
@@ -287,6 +287,25 @@ const commissionCreatedTemplate = ({
   };
 };
 
+// Minimal bounty templates (safe defaults)
+const bountyCreatedTemplate = ({ /* data */ }: { data: unknown }) => ({
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)